src/components/DisplayComments.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from 'react'
+
+function DisplayComments ({children}) {
+    const [showComments, setShowComments] = useState(false);
+
+    const toggleShow = () => {
+        setShowComments((currState) => !currState)     
+    }
+
+    return <div>
+            {showComments ? children : null}
+            <button  onClick={toggleShow}> Show Comments </button>
+    </div>   
```

Comment:
You could maybe put something in here to change the text on the button to be Hide Comments or Show Comments :)